tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -542,7 +542,8 @@ def test_run_needs_uuid(self):
                 run = models.SqlRun()
                 session.add(run)
                 warnings.resetwarnings()
-        assert exception_context.exception.error_code == ErrorCode.Name(INTERNAL_ERROR)
+        assert exception_context.exception.error_code == ErrorCode.Name(BAD_REQUEST), \
+            f"Wrong exception {exception_context.exception.message} raised."
```

Comment:
why do we need `f"Wrong exception {exception_context.exception.message} raised."`? How does the pytest error report look like when this line fails?